ext/BreezeCloudMicrophysicsExt/cloud_microphysics_translations.jl

```diff
@@ -34,15 +34,18 @@ The ``G`` factor combines the effects of thermal conductivity and vapor diffusiv
 on phase change. It appears in the Mason equation for droplet growth:
 
 ```math
-dm/dt = 4π r G 𝒮
+\\frac{dm}{dt} = 4π r G 𝒮
 ```
 
 where ``𝒮`` is supersaturation and ``r`` is droplet radius.
 
 This is a translation of `CloudMicrophysics.Common.G_func_liquid`
 using Breeze's thermodynamics instead of Thermodynamics.jl.
 
-Reference: Eq. (13.28) in Pruppacher & Klett (1997)
+See Eq. (13.28) by [Pruppacher & Klett (1998)](@cite pruppacher1998microphysics).
```

Comment:
@glwagner, what's the proper citation for Pruppacher & Klett? Is this the book I cited?

Reply:
I think its

```
@article{Pruppacher01011998,
author = {Hans R. Pruppacher and James D. Klett and Pao K. Wang},
title = {Microphysics of Clouds and Precipitation},
journal = {Aerosol Science and Technology},
volume = {28},
number = {4},
pages = {381--382},
year = {1998},
publisher = {Taylor \& Francis},
doi = {10.1080/02786829808965531},
URL = {https://doi.org/10.1080/02786829808965531},
eprint = {https://doi.org/10.1080/02786829808965531}
}
```

Reply:
That's just a review of the book?

Reply:
Ah ok, how about this

```
@misc{pruppacher1998microphysics,
  title={Microphysics of clouds and precipitation},
  author={Pruppacher, Hans R and Klett, James D and Wang, Pao K},
  year={1998},
  publisher={Taylor \& Francis}
}
```

Reply:
you can remove the reference if you like

Reply:
> Ah ok, how about this
> 
> ```
> @misc{pruppacher1998microphysics,
>   title={Microphysics of clouds and precipitation},
>   author={Pruppacher, Hans R and Klett, James D and Wang, Pao K},
>   year={1998},
>   publisher={Taylor \& Francis}
> }
> ```

That's what I have in the bibtex at the moment but I think's it's AI-Google mess-up. The book I found is actually by Kluwer Publishing. The Taylor & Francis is the book review. And I doubt Wang is involved in any way other than the book review.

Reply:
Let's leave it! I'll just put the proper bibtex entry for the book. It's always good to have references if they are the right ones!

Reply:
how about this:

```
@book{pruppacher1996microphysics,
  title={Microphysics of Clouds and Precipitation},
  author={Pruppacher, HR and Klett, JD},
  volume={18},
  year={1996},
  publisher={Springer Science \& Business Media}
}
```

Reply:
> how about this:
> 
> ```
> @book{pruppacher1996microphysics,
>   title={Microphysics of Clouds and Precipitation},
>   author={Pruppacher, HR and Klett, JD},
>   volume={18},
>   year={1996},
>   publisher={Springer Science \& Business Media}
> }
> ```

Okie! But the 2nd edition 2010. [22988c7](https://github.com/NumericalEarth/Breeze.jl/pull/345/commits/22988c746c33ac8f77adf697f3a4d2d619644dce) deals with this.